packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -150,37 +150,53 @@ function createSoapBodyContent({
               },
               [`${prefix}semanticsText`]: "LivingSubject.administrativeGender",
             },
-            [`${prefix}livingSubjectBirthTime`]: {
-              [`${prefix}value`]: {
-                "@_value": patientBirthtime,
-              },
-              [`${prefix}semanticsText`]: "LivingSubject.birthTime",
-            },
-            [`${prefix}livingSubjectName`]: {
-              [`${prefix}value`]: {
-                [`${prefix}family`]: patientFamilyName,
-                [`${prefix}given`]: patientGivenName,
-              },
-              [`${prefix}semanticsText`]: "LivingSubject.name",
-            },
-            [`${prefix}patientAddress`]: patientAddress
+            [`${prefix}livingSubjectBirthTime`]: patientBirthtime
               ? {
                   [`${prefix}value`]: {
-                    [`${prefix}streetAddressLine`]: patientAddress.line?.join(", "),
-                    [`${prefix}city`]: patientAddress?.city,
-                    [`${prefix}state`]: patientAddress?.state,
-                    [`${prefix}postalCode`]: patientAddress?.postalCode,
-                    [`${prefix}country`]: patientAddress?.country,
+                    "@_value": patientBirthtime,
                   },
+                  [`${prefix}semanticsText`]: "LivingSubject.birthTime",
+                }
+              : {},
+            [`${prefix}livingSubjectIdentifier`]: identifiers
+              ? {
+                  [`${prefix}value`]: identifiers.map(identifier => ({
+                    "@_extension": identifier.value,
+                    "@_root": identifier.system,
+                  })),
+                  [`${prefix}semanticsText`]: "LivingSubject.id",
+                }
+              : {},
+            [`${prefix}livingSubjectName`]: patientNames
+              ? {
+                  [`${prefix}value`]: patientNames.map(name => ({
```

Comment:
No. The xml parser handles this and it will result being 
this key value pair:

"livingSubjectName": {}

which becomes <livingSubjectName/> in the resulting xml

Reply:
https://github.com/metriport/metriport/pull/2143#pullrequestreview-2081342371